plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/MockAmazonS3.java

```diff
@@ -45,8 +45,12 @@
 public class MockAmazonS3
         extends AbstractAmazonS3
 {
+    public static final String NO_SUCH_KEY_ERROR_CODE = "NoSuchKey";
+    public static final String NO_SUCH_BUCKET_ERROR_CODE = "NoSuchBucket";
```

Comment:
This is used in `TestTrinoS3FileSystem` only.
Move the constant to TrinoS3FileSystem and make it package-private.
Use it in ``throwIfFileNotFound``